tests/entities_column_value_simple_tests.py

```diff
@@ -103,8 +103,92 @@ def test_should_throw_exception_when_setting_an_invalid_value():
     eq_(column_value.value, str(text))
 
 
+def test_should_create_a_column_value_with_no_API_input_data():
+
+    # Arrange
+
+    id = 'people_1'
+    column_type = ColumnType.people
+    title = 'people 1'
+    value = None
+    
+    column_value = en.cv.create_column_value(column_type, id=id, title=title,value=value)
+    
+    # Act
+    format = column_value.format()
+
+    # Assert
+
+    eq_(format, {})
+
+def test_should_create_a_column_value_with_no_API_input_data():
+
+    # Arrange
+
+    id = 'people_1'
+    column_type = ColumnType.people
+    title = 'people 1'
+    # value = [{'id':134 , 'kind':PeopleKind.person},
+    #         {'id': 234 , 'kind':PeopleKind.person}]
+    
+    column_value = en.cv.create_column_value(column_type, id=id, title=title)
+    
+    # Act
+    format = column_value.format()
+
+    # Assert
+
+    eq_(format, {})
+
+def test_should_create_a_column_value_with_API_input_data():
+
+    # Arrange
+
+    id = 'people_1'
+    column_type = ColumnType.people
+    title = 'people 1'
+    value_dict ={'personsAndTeams': [{'id':134 , 'kind':'person'},{'id':11234 , 'kind':'person'}]}
+    value = json.dumps(value_dict)
+    column_value = en.cv.create_column_value(column_type, id=id, title=title,value=value)
+    
+    # Act
+    person_value = column_value.format()['personsAndTeams']
+    format = person_value[0]
+
+    # Assert
+
+    eq_(format['id'],134)
+
+def test_should_set_value_to_none():
+     # Arrange
+
+    id = 'people_1'
+    column_type = ColumnType.people
+    title = 'people 1'
+    column_value = en.cv.create_column_value(column_type, id=id, title=title)
+    
+    # Act
+    column_value.value = []
+    format = column_value.format()
+
+    # Assert
+
+    eq_(format,{})
+
+@raises(e.ColumnValueError)
+def test_should_throw_exception_when_setting_an_invalid_column_value():
```

Comment:
Please rename to test_should_throw_an_exception_when_setting_an_invalid_value_to_people_column_value.